src/modules/common/components/bars/topBar/hardwareWallet.css

```diff
@@ -0,0 +1,154 @@
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+
+  & .hwIcon {
+    width: 20px;
+    height: 100% !important;
+    background-color: transparent;
+    padding: 0px 6px !important;
+    max-width: fit-content;
+    min-width: fit-content;
+    position: relative;
+
+    &.connected::after {
+      background-color: var(--color-jade-green);
+    }
+
+    &.disconnected::after {
+      background-color: red;
+    }
+
+    &.standby::after {
+      background-color: var(--color-warning-border);
+    }
+
+    &::after {
+      width: 6px;
+      height: 6px;
+      border-radius: 50%;
+      content: "";
+      display: block;
+      position: absolute;
+      margin-top: -25px;
+      margin-left: 14px;
+    }
+
+    & > img {
+      margin-top: 5px;
+    }
+  }
+
+  & > button ~ div {
+    margin-right: -23px;
+    margin-top: -18px;
+    width: 229px;
+  }
+}
+
+.dropdown {
+  width: 200px;
+  padding: 24px 16px 17px 16px;
+  border-radius: var(--border-radius-box);
+
+  & h6 {
+    font-family: var(--heading-font) !important;
+    font-size: var(--font-size-h6) !important;
+    color: var(--color-content-light) !important;
+    padding: 0px !important;
+    margin: 0px;
+
+    &:hover {
+      background-color: transparent !important;
+    }
+  }
+
+  & h6 ~ ul {
+    padding: 0px !important;
+    font-size: var(--paragraph-font-size-s) !important;
+    margin: 6px 0px 0px 0px;
+
+    &:hover {
+      background-color: transparent !important;
+      color: var(--color-slate-gray);
+    }
+
+    & > li {
+      display: flex;
+      flex-direction: row;
+      margin-top: 10px;
+    }
+
+    & > li > div {
+      width: 35%;
+      color: var(--color-slate-gray);
+    }
+
+    & > li > div:nth-child(2) {
+      width: 65%;
+      color: var(--color-content-light);
+    }
+  }
+}
+
+.statusWrapper {
+  border-radius: 5px;
+  border: 1px solid transparent;
+  display: flex;
+  align-items: center;
+  font-size: var(--button-font-size-s);
+  width: fit-content;
+  padding: 0px 8px;
+  text-transform: capitalize;
+
+  &::before {
+    width: 6px;
+    height: 6px;
+    border-radius: 50%;
+    content: "";
+    display: block;
+    margin-right: 8px;
+  }
+
+  &.standby {
+    border-color: var(--color-warning-border-transparent);
+    background-color: var(--color-warning-border-background-transparent);
+    color: var(--color-warning-border) !important;
+
+    &::before {
+      background-color: var(--color-warning-border);
+    }
+  }
+
+  &.connected {
+    border-color: var(--color-jade-green-border);
+    background-color: var(--color-jade-green-background);
+    color: var(--color-jade-green) !important;
+
+    &::before {
+      background-color: var(--color-jade-green);
+    }
+  }
+
+  &.disconnected {
+    border-color: var(--color-red-border);
+    background-color: var(--color-red-background);
+    color: var(--color-red) !important;
+
+    &::before {
+      background-color: var(--color-red);
+    }
+  }
+}
```

Comment:
@eniolam1000752 Since the icon sin't shown when the HW wallet is disconnected, then there's no need for this style